src/com/teamfivetech/passwordmanager/controller/PasswordManager.java

```diff
@@ -58,7 +60,7 @@ private void listPasswords() {
         try {
             readLogins = loginIO.read();
         } catch (IOException e) {
-            e.printStackTrace();
+            e.getMessage();
```

Comment:
getMessage() returns a String. but you aren't doing anything with it. you can wrap this in a sout